manifests/gcp_marketplace/guide.md

```diff
@@ -59,11 +59,8 @@ and store the service account credential as a Kubernetes secret `user-gcp-sa` in
 # Create credential for the service account
 gcloud iam service-accounts keys create application_default_credentials.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
 
-# Make sure the secret is created under the correct namespace.
-kubectl config set-context --current --namespace=$NAMESPACE
-
 # Attempt to create a k8s secret. If already exists, override.
-kubectl create secret generic user-gcp-sa \
+kubectl create secret generic user-gcp-sa -n $NAMESPACE\
```

Comment:
Minor usability concern, copying a command of 3 lines and then modifying something on the first line is hard.
I prefer
```
kubectl create secret generic user-gcp-sa \
--from-file=user-gcp-sa.json=application_default_credentials.json \
-n $NAMESPACE --dry-run -o yaml  |  kubectl apply -f -
```

Reply:
Done.